doc/usermanual/darkroom/modules/color/lut3d.xml

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE sect3 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+               "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd" [
+		<!ENTITY % darktable_dtd SYSTEM "../../../dtd/darktable.dtd">
+		%darktable_dtd;
+		]>
+<sect3 status="draft" id="lut3d">
+
+  <title>3D LUT</title>
```

Comment:
GENERAL COMMENT: this  is my first documentation review on Darktable; I tend to make a fair amount of comments/suggestions/nitpicking. They're obviously just that, feel free to disregard them. If you need me to back off on the amount of suggestions, please let me know.

Also: English is NOT my native language, so there's a possibility I correct things that should not be corrected, or that I let things be that should be. 

</caveat> :)

Reply:
no problem.